contributing.md

```diff
@@ -0,0 +1,34 @@
+# Contribution Guidelines
+Ensure your pull request adheres to the following guidelines
+- Leave issues if you have any interesting ideas or bugs interesting ideas or bugs.
+- Use English in all cases(Communication, Comment, Code review...)
+
+## Code contribution guide
+- Consist indentation of 4 spaces.
+- File name uses snake case for file name.
+- Variable name uses pascal case for react components and camel case for others.
```

Comment:
Please fix it

Reply:
`- Name a react component variable as pascal case, others camel case.` Done!